content/en/docs/tasks/observability/metrics/customize-metrics/index.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Customizing Istio Metrics
+description: This task shows you how to customize the Istio metrics.
+weight: 25
+keywords: [telemetry,metrics,customize]
+---
+
+This task shows you how to customize the metrics that Istio generates.
+
+Istio generates telemetry that various dashboards consume to help you visualize
+your mesh. For example, dashboards that support Istio include:
+
+* [Grafana](/docs/tasks/observability/metrics/using-istio-dashboard/)
+* [Kiali](/docs/tasks/observability/kiali/)
+* [Prometheus](/docs/tasks/observability/metrics/querying-metrics/)
+
+By default, Istio defines and generates a set of standard metrics (e.g.
+`requests_total`), but you can also customize them and create new metrics.
+
+## Custom statistics configuration
+
+Istio uses the Envoy proxy to generate metrics and provides its configuration in
+the `EnvoyFilter` at
+[`manifests/charts/istio-control/istio-discovery/templates/telemetryv2_1.6.yaml`]({{<github_blob>}}/manifests/charts/istio-control/istio-discovery/templates/telemetryv2_1.6.yaml).
+
+Configuring custom statistics involves two sections of the
+`EnvoyFilter`: `definitions` and `metrics`. The `definitions` section
+supports creating new metrics by name, the expected value expression, and the
+metric type (`counter`, `gauge`, and `histogram`). The `metrics` section
+provides values for the metric dimensions as expressions, and allows you to
+remove or override the existing metric dimensions. You can modify the standard
+metrics using `tags_to_remove` or by re-defining a dimension.
+
+## Before you begin
+
+[Install Istio](/docs/setup/) in your cluster and deploy an application.
+Alternatively, you can set up custom statistics as part of the Istio
+installation.
+
+## Enable custom metrics
+
+Edit the the `EnvoyFilter` to add or modify dimensions and metrics. Then, add
+annotations to all the Istio-enabled pods to extract the new or modified
+dimensions.
+
+1. Locate the `envoy.wasm.stats` extension configuration. The default
```

Comment:
Locate how? EnvoyFilter with that name? I don't see one in the template

Reply:
This got updated to `istio.stats`. Sorry this is all in flux, and in fact, I'll need to make more edits after the first iteration.

Reply:
@mandarjog : should we just keep it at envoy.wasm.stats only as networking stats filter name is still "envoy.filters.network.wasm" only?